app/assets/stylesheets/nav.scss

```diff
@@ -9,31 +9,59 @@ $button-color: #1E90FF;
   margin-left: 5%;
   list-style-type: none;
 }
-
+.top-bar {
+  background-color: #3f3f3f;
+}
+h3{
+  font-family: $font-sans;
+}
+h2{
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.